src/main/java/com/example/Main.java

```diff
@@ -1,7 +1,21 @@
 package com.example;
 
+/**
+ * Main class that contains void main
+ *
+ * @author Elena Yevsyukova
+ * @version 1.1
+ */
 public class Main {
+    /**
+     * Here starts point of the program
+     *
+     * @param args command line values
```

Comment:
I know this from the convention about the `main()` method. What parameters does it accept? How do I use them?